docs/content/configuration/_index.md

```diff
@@ -124,6 +124,7 @@ weight: 2
 |    --enabled-proxy-protocol                 | enable proxy protocol | false | PROXY_ENABLE_PROXY_PROTOCOL
 |    --max-idle-connections value             | max idle upstream / keycloak connections to keep alive, ready for reuse | 0 | PROXY_MAX_IDLE_CONNS
 |    --max-idle-connections-per-host value    | limits the number of idle connections maintained per host | 0 | PROXY_MAX_IDLE_CONNS_PER_HOST
+|    --server-grace-timeout value             | the server graceful period before shutdown | 10s | PROXY_SERVER_READ_TIMEOUT
```

Comment:
you have here PROXY_SERVER_READ_TIMEOUT, which is not right

Reply:
Opsy! Fixed